thirdparty/github.go

```diff
@@ -1026,16 +1026,6 @@ func GetMergeablePullRequest(ctx context.Context, issue int, githubToken, owner,
 		return nil, errors.Wrap(err, "GitHub returned an incomplete PR")
 	}
 
-	if pr.Mergeable == nil {
-		if *pr.Merged {
-			return pr, errors.New("PR is already merged")
-		}
-		// GitHub hasn't yet tested if the PR is mergeable.
-		// Check back later
-		// See: https://developer.github.com/v3/pulls/#response-1
-		return pr, errors.New("GitHub hasn't yet generated a merge commit")
-	}
-
```

Comment:
L1039 needs to check if pr.Mergeable is nil before dereferencing.

Reply:
thanks! switched to utility.FromBoolPtr.

Reply:
I think it has to be `utility.FromBoolTPtr` because if the pointer is nil, we're assuming it's mergeable.

Reply:
It seems to me that if the pointer is nil, FromBoolPtr returns false, so we end up with !(false), so we enter the if and return errors.New("PR is not mergeable"). 

Am I misreading it?

Reply:
I thought the point of removing the nil check was to assume that if the mergeability check isn't available yet, we just skip the check, and would only want to exit early if `*Mergeable` was true. If `nil` (i.e. can't check mergeability yet) is equivalent to `false`, then the if statement will make it claim that it's not mergeable, but we actually don't know if it's mergeable.

Reply:
ah yes :see_no_evil: . Thanks! fixed!